interfaces/builtin/network_bind_test.go

```diff
@@ -37,16 +39,29 @@ var _ = Suite(&NetworkBindInterfaceSuite{
 })
 
 func (s *NetworkBindInterfaceSuite) SetUpTest(c *C) {
-	s.slot = &interfaces.Slot{
-		Snap:      "ubuntu-core",
-		Name:      "network-bind",
-		Interface: "network-bind",
-	}
-	s.plug = &interfaces.Plug{
-		Snap:      "snap",
-		Name:      "network-bind",
-		Interface: "network-bind",
-	}
+	info1, err := snap.InfoFromSnapYaml([]byte(`
```

Comment:
For such trivial cases, I'd probably use logic closer to the original code here, with an interfaces.Slot inlined. Shorter and more clear.